src/utils/storage/local.ts

```diff
@@ -0,0 +1,20 @@
+const PREFIX = 'SAFE__'
+
+const prefixKey = (key: string): string => `${PREFIX}${key}`
+
+export const getItem = <T>(key: string): T | undefined => {
```

Comment:
Why are you not using the functions we already have for storage interaction?

Reply:
Which ones?

Reply:
`loadFromStorage()` and `saveToStorage()` at `src/utils/storage/index.ts`

Reply:
I created some for the `sessionStorage` at `src/utils/storage/session.ts` that don't rely on ImmortalDB. They might be of help.

Reply:
I'll merge session and local, they share 99% of the code.